hack/arktos-up.sh

```diff
@@ -400,7 +400,10 @@ Alternatively, you can write to the default kubeconfig:
   cluster/kubectl.sh config set-credentials myself ${AUTH_ARGS}
   cluster/kubectl.sh config set-context local --cluster=local --user=myself
   cluster/kubectl.sh config use-context local
+  cluster/kubectl.sh config get-contexts
```

Comment:
Minimal changes needed. There is no need to update arktos-up.sh just for the echo message.